library/src/main/java/eightbitlab/com/blurview/BlockingBlurController.java

```diff
@@ -287,12 +287,7 @@ void setBlurEnabledInternal(boolean enabled) {
 
     @Override
     public BlurViewFacade setBlurEnabled(final boolean enabled) {
-        blurView.post(new Runnable() {
-            @Override
-            public void run() {
-                setBlurEnabledInternal(enabled);
-            }
-        });
+        setBlurEnabledInternal(enabled);
```

Comment:
`internal*` methods can be removed and inlined, as they are now doing the same stuff